swagger/v1/api.json

```
@@ -550,6 +590,50 @@
     }
   ],
   "paths": {
+    "/api/v1/groups": {
+      "get": {
+        "summary": "Lister les groupes (représentation des territoires)",
+        "tags": [
+          "Groups"
```

```suggestion
          "Group"
```

je pense une typo qui destructure la mise en page
ou alors renommer le tag en ligne 574 avec `Groups`

je pense que le singulier est mieux adapté

---

Je suis d'accord 👍